### PR TITLE
tab to space all html match

### PR DESCRIPTION
**Pull request by loanfedora**

### CODE REVIEW

**Comment by loanfedora** [comment]
can this be accepted @etix?

File: templates/base.html [hunk 1]
```diff
@@ -1,140 +1,140 @@
...
+                text-decoration: underline;
+            }
+        </style>
 {{template "head" .}}
-	</head>
-	<body>
-		<div id="wrapper">
-			<div id="header">
-				<div class="title">
-					<a href="http://github.com/etix/mirrorbits" target="_blank" style="text-decoration: none; color: inherit;">Mirrorbits <i class="fa fa-globe" style="vertical-align: middle;" aria-hidden="true"></i></a>
```
Comment (lfauvet-bc) [inline templates/base.html:127]: Please keep `https` here.

**Review by lfauvet-bc — CHANGES_REQUESTED** [review]

File: templates/base.html (hunk 1 above)
Comment (loanfedora) [reply templates/base.html:127]: currently your master branch is set to http @ <https://github.com/etix/mirrorbits/blob/master/templates/base.html#L127/>

this makes it https @ https://github.com/etix/mirrorbits/pull/119/files#diff-690b81fad8df2a1f1ce37c846641d9247e3472d244f17039d2910a5ed0c98d5eR127

Comment (loanfedora) [reply templates/base.html:127]: plz review again @lfauvet-bc

Comment (loanfedora) [reply templates/base.html:127]: plz review again @lfauvet-bc  @etix

**Comment by etix** [comment]
Thanks for your contribution.

**Pull request merged**